### PR TITLE
Twillio logic fixed within the AlertForm

### PR DESCRIPTION
**Pull request by justinnoah**
fixes sending out alerts via text

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -67,5 +67,17 @@ def seed_db():
...
 
+@manager.command
+def twillio_test():
+    """Quick sms test to make sure twillio is functioning as expected."""
+    from _15thnight.twilio_client import send_sms
+
+    with app.app_context():
+        # This default test number will force twillio to fail, but still be
+        # a valid api call
+        phone_number = app.config.get("TWILIO_TEST_NUMBER", "+15555555555")
```
Comment (nobane) [inline manage.py]: This should take in the phone number from the command line, then fall back to the config.

File: configdist.py [hunk 2]
```diff
@@ -1,17 +1,18 @@
 # Database config
 DATABASE_URL = "sqlite:///test.db"
 
-TWILIO_ACCOUNT_SID = ''
-TWILIO_ACCOUNT_AUTH_TOKEN = ''
-TWILIO_FROM_NUMBER = ''
+TWILIO_ACCOUNT_SID = ""
+TWILIO_ACCOUNT_AUTH_TOKEN = ""
+TWILIO_FROM_NUMBER = ""
+TWILIO_TEST_NUMBER = ""
```
Comment (nobane) [inline configdist.py]: Should probably put in a comment here about how the number needs to be `+1XXXXXXXXXX`.

**Comment by justinnoah** [comment]
@spectralsun fixed per your suggestions

File: manage.py [hunk 3]
```diff
@@ -67,5 +67,22 @@ def seed_db():
     User('admin@test.com', '1234', '5415551234', [], 'admin').save()
 
 
+@manager.command
+@manager.option('-n', '--number', help='Twillio Test Phone Number')
```
Comment (nobane) [inline manage.py]: Help should have text that says the number should be in +1XXXXXXXXXX.

File: manage.py [hunk 4]
```diff
@@ -67,5 +67,23 @@ def seed_db():
     User('admin@test.com', '1234', '5415551234', [], 'admin').save()
 
 
+@manager.command
+@manager.option(
+    '-n', '--number', help='Twillio Test Phone Number (e.g. 1234567890)')
```
Comment (nobane) [inline manage.py]: Make this number `8885550000` please.

File: _15thnight/__init__.py [hunk 5]
```diff
@@ -59,6 +54,17 @@ def shutdown_session(response):
...
 
+if not app.config.get('DEBUG'):
+    @app.errorhandler(404)
+    @app.errorhandler(Exception)
+    def error_page(error):
+        """Generic Error handling."""
+        code = 500
+        if isinstance(error, HTTPException):
+            code = error.code
+        return render_template("error.html", error_code=code), code
```
Comment (nobane) [inline _15thnight/__init__.py]: This code was removed, please remove.

**Comment by nobane** [comment]
+1

**Pull request merged**